.github/workflows/changelog.yaml

```diff
@@ -0,0 +1,32 @@
+name: Build, generate changelog & publish
+
+on:
+  push:
+    branches: [master]
+
+jobs:
+  release:
+    runs-on: ubuntu-latest
+    timeout-minutes: 15
+    env:
+      NPM_PACKAGE_ACCESS: ${{ secrets.NPM_PACKAGE_ACCESS }}
+      NODE_AUTH_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+    steps:
+      - uses: actions/checkout@v2
+      - uses: fregante/setup-git-user@v1
+      - uses: actions/setup-node@v1
+        with:
+          node-version: 14
+          registry-url: "https://npm.pkg.github.com/"
+      - name: Install dependencies
+        run: npm ci
+      - name: Get latest commits
+        run: git pull
+      - name: Build
+        run: npm run build
+      - name: Generate changelog
+        run: npm run release
+      - name: Commit and push tag
+        run: git push --follow-tags origin master
```

Comment:
ce projet etant encore "in progress" peu etre mieux d'attendre avant d'auto la release ici.